lib/server/connect.js

```diff
@@ -40,6 +41,9 @@ Meteor.methods({
       data.times.push(ev);
     });
 
+    // set unique id for each server session
+    data._id = serverId;
```

Comment:
Also write a test case and check you are sending a server id as well. 
May be tests might fail. (Run tests locally, travis is failing. That's fine)